### ISSUE TITLE
Notifications are not announced by VoiceOver

### ISSUE THREAD

**Issue by juuso-vaadin** [vaadin/vaadin-notification#113]
### Description
Vaadin Notifications are not announced by VoiceOver. The component has `aria-live="polite"` in place but it doesn't get picked up.

### Solution proposal
See "Invisible live regions"  in https://inclusive-components.design/notifications/

### Browsers Affected
Tested with macOS VoiceOver using Chrome and Safari

**Comment by web-padawan** [comment]
We considered using `aria-live` in the avatar group and here is some information: https://github.com/vaadin/vaadin-avatar/issues/44#issuecomment-658001219

I would suggest that we change the implementation of `vaadin-notification` to use `iron-a11y-announcer` which is tested  using VoiceOve (Chrome and Safari) and NVDA, and confirmed to work: https://github.com/vaadin/vaadin-avatar/issues/44#issuecomment-679085371
